### PR TITLE
Laboratory Work #1, Ivanov Artur - 20FPL1

### PR DESCRIPTION
**Pull request by ArturIvanov228**

### CODE REVIEW

File: scrapper_config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "seed_urls": [],
-    "total_articles_to_find_and_parse": 0
+    "total_articles_to_find_and_parse": 7
```
Comment (dmitry-uraev) [inline scrapper_config.json:3]: its right time to fill config file and start implementing scrapper.py

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]

File: scrapper.py [hunk 2]
```diff
@@ -25,39 +36,135 @@ class Crawler:
...
+        self.seed_urls = seed_urls
+        self.total_max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        cells = article_bs.find_all("div", class_='col-lg-3 col-sm-6')
+        for cell in cells:
+            if len(self.urls) >= self.total_max_articles:
+                break
```
Comment (dmitry-uraev) [inline scrapper.py:49]: can you rewrite this function without break point?

File: scrapper.py [hunk 3]
```diff
@@ -25,39 +36,135 @@ class Crawler:
...
+            self.urls.append('https://pravdasevera.ru' + link['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed_url in self.seed_urls:
+            if len(self.urls) >= self.total_max_articles:
+                break
```
Comment (dmitry-uraev) [inline scrapper.py:59]: same as above

File: scrapper.py [hunk 4]
```diff
@@ -25,39 +36,135 @@ class Crawler:
...
+            link = cell.find('a')
+            self.urls.append('https://pravdasevera.ru' + link['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed_url in self.seed_urls:
+            if len(self.urls) >= self.total_max_articles:
```
Comment (dmitry-uraev) [inline scrapper.py:58]: you have two same conditions in two functions - can we use only one condition?

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Good, please fix comments

File: scrapper.py (hunk 2 above)
Comment (ArturIvanov228) [reply scrapper.py]: solved this problem

File: scrapper.py (hunk 3 above)
Comment (ArturIvanov228) [reply scrapper.py]: solved this problem

File: scrapper.py (hunk 4 above)
Comment (ArturIvanov228) [reply scrapper.py]: Yes

File: config/target_score.txt [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
 
 # Target score for pipeline.py:
 0
```
Comment (dmitry-uraev) [inline config/target_score.txt:5]: It is time to implement the second lab as well

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]

File: config/target_score.txt (hunk 5 above)
Comment (ArturIvanov228) [reply config/target_score.txt:5]: sorry, I won't be able to do the second job. I will do it on holidays not for evaluation

**Pull request closed without merging**